lib/evaluate/fold-integer.cpp

```diff
@@ -630,6 +630,32 @@ std::optional<std::int64_t> ToInt64(const Expr<SomeType> &expr) {
   }
 }
 
+std::optional<std::string> ToString(const Expr<SomeCharacter> &expr) {
```

Comment:
None of these functions have anything to do with integers; `fold-integer.cpp` seems like a confusing home for them.

Reply:
Thanks Klausler, I have moved them to fold-character.cpp and fold-logical.cpp files..